packages/react-components/jest.config.ts

```diff
@@ -24,6 +24,9 @@ const config = {
     '<rootDir>/src/components/knowledge-graph/KnowledgeGraphPanel.tsx',
     '<rootDir>/src/components/chart/trendCursor/mouseEvents/useTrendCursorsEvents.ts',
     '<rootDir>/src/components/chart/events/useHandleChartEvents.ts',
+    '<rootDir>/src/echarts/extensions/trendCursors/view/*',
```

Comment:
any particular reason we are not testing these files ?

Reply:
They are echarts specific classes which are difficult to work with in a unit test. They are covered in the e2e test in dashboard.